### ISSUE TITLE
much Gratitude~ (thx for push to play store ✌️💝✊

### ISSUE THREAD

**Issue by ipfreer** [jazzm0/ssh-daemon#36]
I hope not to have bothered you with making issue, I pray you are well and looking forward to new year ahead🙏

I simply want to thank you, sincerely and from heart, as i was growing very concerned you were leaving Play store beyond for good, like a few other excellent open source projects have this past year due new rules on permissions, privacy declarations, code cleanup, etc..

I found your expert level and very excellent ssh server some months back near summer when my phone was dying and I needed trustworthy non rooted ways to get things moved, fast, that didn't feel invasive like oneplus clone new phone app nor the goog's seemingly very selective backups and/or transfer/move features and app.

then I started having issue with your server no longer work in Oct when upgraded to 11, then 12, and now 13 so fast for that phone I was going to buy from friend oneplus 9pro.

I finally installed your server from Fdroid, which I don't really want to do, I mean I love them guys and owe them even more gratitude help me over the years, but have migrated mind to desire app author signed releases first and foremost (prefer from repo direct, but need app to help manage keep updated), or Play store version which I feel my whole life is again open up to the GOOG, and they getting picky on alt signed apps, like team Fdroid does.

now droid-ify is my savior on app update and management, since they give option to install version from author repo/github or fdroid, and I choose author signed every time. then seeing many updates made on your app over past few months but no apk, and no update to store even I'm on beta tester team there, and now finally get new phone, not one plus nine pro it's just not for me, but friend let me try their pixel 7 pro and 😳🤯💯🥰 what a great phone! it even feels very secure and seems goog even now respect privacy and the human operating Android now with giving ability to control apps, and permissions, and tons of new stuff used to be root only access, that my brain is still such at vs10 and my real old samsung that should have been replaced years ago, before breaking point anyways😂

I made github account to post issues, but too wordy always so never do, so I subscribe and follow and keep tabs with octodroid app (also awesomesauce f/oss😍

I keep wanting to ask here for playstore update, but never do. now I see you did this latest vs .13 even!  now I can get back to signed by author version, and autoupdates again, yay!! *⁠\⁠0⁠/⁠*

so I had to come here and thank you, and make promise to start sponsor by Spring (i pay off Casita and shop in two months!!🤯🥳🙏🤠. and your project is one of a few that I want to sponsor with some monies because you really help me out and I truly appreciate and I know words, even kind and sincere, don't put good food on your table💝

sorry this is list as issue, it is ok to close immediately if annoying or wrong to do. it's also ok to leave open, or pin it even 🫣🤫🤭 to encourage others to sponsor their favorite OSS projects, and also remind me my promise to do so by Spring
😶‍🌫️🌱 🌼🌻🏵️🌵🧙🏼‍♂️🌵

much respect to you and yours, 
✌️and 🫶

signed, old broken toy 🪖
iPfreer✓   °°•∆

**Comment by jazzm0** [comment]
Thanks,actually that is the first appreciation ever! People just demand and demand, but a little thank you is too much for most. Really happy for your words!

thanks
